classes/PodsI18n.php

```diff
@@ -196,4 +205,240 @@ private static function default_strings() {
 
 	}
 
+	/**
+	 * Get current language information from Multilingual plugins
+	 *
+	 * @since 2.6.6
+	 * @since 2.7 Moved to this class from PodsAPI
+	 *
+	 * @param array $args (optional) {
+	 *     @type bool $redo Rerun logic?
+	 * }
+	 *
+	 * @return array
+	 */
+	public static function get_current_language( $args = array() ) {
```

Comment:
We have `$current_language` and a `$current_language_data` members, this method returns the language data.  Should the method reflect that with the name `get_current_language_data()`?